src/reviews/reviews.service.ts

```diff
@@ -30,15 +30,15 @@ export class ReviewsService {
   }
 
   async findOne(id: number): Promise<any> {
-    const review = await this.prisma.review.findUnique({where: {id}});
+    const review = await this.prisma.review.findUnique({ where: { id } });
     if (!review) {
       throw new NotFoundException(`Review with id ${id} not found`);
     }
     return review;
   }
 
   async create(createReviewDto: CreateReviewDto) {
-    await this.prisma.review.create({data: {...createReviewDto}});
+    await this.prisma.review.create({ data: { ...createReviewDto } });
```

Comment:
Variable name Dto denotes it's an interface. So, rename this variable to `createdReview`. This increases understandability of the code.